readthedocs/projects/models.py

```diff
@@ -114,6 +113,16 @@ class Project(models.Model):
         help_text=_("Google Analytics Tracking ID (ex. UA-22345342-1). "
                     "This may slow down your page loads."))
 
+    # Sphinx specific build options.
```

Comment:
No it's really straightforward to rename the field again. It should be a simple search&replace. I will do so on Thursday.